src/edu/ucsb/nceas/metacat/admin/upgrade/solr/SolrConfigUpgrader.java

```diff
@@ -69,29 +53,31 @@ public SolrConfigUpgrader() throws PropertyNotFoundException, ServiceException {
             solrHomePath = PropertyService.getProperty("solr.homeDir");
             String indexContext = PropertyService.getProperty("index.context");
             currentHash = PropertyService.getProperty("index.configFile.current.hash");
-            logMetacat.info("the current hash is ================== "+currentHash);
+            logMetacat.info("SolrConfigUpgrader.constructor - the current hash is " + currentHash);
             String metacatWebInf = ServiceService.getRealConfigDir();
-            metacatIndexSolrHome = metacatWebInf + "/../../" + indexContext + "/WEB-INF/classes/solr-home";
+            metacatIndexSolrHome = metacatWebInf + "/../../" + indexContext
+                                                                    + "/WEB-INF/classes/solr-home";
         } catch (PropertyNotFoundException pnfe) {
-            throw new PropertyNotFoundException("SolrConfigUpdator.Constructor - could not get a metacat property in the metacat.properties file - "
+            throw new PropertyNotFoundException("SolrConfigUpdator.Constructor - "
```

Comment:
SolrConfigUpdator is wrong class name. See earlier comment about removing class & method names from log messages